src/tennisSimulator/Player.java

```diff
@@ -9,7 +9,7 @@ public class Player implements Comparable<Player> {
     private boolean isServing = false;
     private int gamesWon = 0;
     private int setsWon = 0;
-    private Object handsWon = 0;
+    private int handsWon = 0;
```

Comment:
Maybe instead of counting how many hands are won, have specific status, like in real tennis game (15, 30, 40, A, GAME), which would determine status of a game